src/styles/main.scss

```diff
@@ -21,24 +21,24 @@ body {
   left: 50%;
   transform-origin: bottom center;
   transform: translateX(-50%) translateY(-100%);
-}
 
-.hand-second {
-  width: 2px;
-  height: 50%;
-  background-color: red;
-  animation: rotate360 5s linear 1;
-}
+  &--second {
+    width: 2px;
+    height: 50%;
+    background-color: #f00;
+    animation: rotate360 5s linear 1;
+  }
 
-.hand-minute {
-  width: 4px;
-  height: 50%;
-  background-color: blue;
-  animation: rotate360 60s linear 5;
+  &--minute {
+    width: 4px;
+    height: 50%;
+    background-color: #000;
+    animation: rotate360 60s linear 5;
+  }
```

Comment:
Add @mixin to avoid repeating

![image](https://github.com/mate-academy/layout_stop-watch/assets/122356496/9598ffe7-bde8-4133-a349-988cf29aaf39)
![image](https://github.com/mate-academy/layout_stop-watch/assets/122356496/54a0b9ff-9c68-4c67-941c-3ba1947b66d5)